pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -258,435 +259,447 @@ private BrokerResponseNative handleRequest(long requestId, String query,
       throws Exception {
     LOGGER.debug("SQL query for request {}: {}", requestId, query);
 
-    long compilationStartTimeNs;
-    PinotQuery pinotQuery;
+    Tracing.ThreadAccountantOps.setupRunner(String.valueOf(requestId));
```

Comment:
Very good question! Since the handling of in subquery will be in the same thread, so it's okay to call the setup once and there will only be one runner entry with the unique request ID.